tests/test_create_template.py

```diff
@@ -41,11 +41,11 @@ def test_run_cookiecutter_and_plugin_tests_with_napari_prefix(cookies, capsys):
 
     assert result.exit_code == 0
     assert result.exception is None
-    assert result.project.basename == "napari-foo"
-    assert result.project.isdir()
-    assert result.project.join("src").isdir()
-    assert result.project.join("src", "napari_foo", "__init__.py").isfile()
-    assert result.project.join("src", "napari_foo", "_tests", "test_reader.py").isfile()
+    assert result.project_path.name == "napari-foo"
+    assert result.project_path.is_dir()
+    assert result.project_path.joinpath("src").is_dir()
+    assert result.project_path.joinpath("src", "napari_foo", "__init__.py").is_file()
+    assert result.project_path.joinpath("src", "napari_foo", "_tests", "test_reader.py").is_file()
```

Comment:
I don't understand these changes but I'm ok with them...

Reply:
the `project` attribute of `cookies.bake` is deprecated and should be replaced with `project_path`. 

It looks like I put two changes in one PR. Will update description